api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -173,4 +177,92 @@ public void deleteOrderGroupAttributeType_shouldDeleteOrderGroupAttributeTypeFro
 		dao.deleteOrderGroupAttributeType(orderGroupAttributeType);
 		assertNull(dao.getOrderGroupAttributeByUuid(UUID));
 	}
+
+	/**
+	 * @see {@link HibernateOrderDAO#getAllOrderAttributeTypes()}
+	 * @throws Exception
+	 */
+	@Test
+	public void getAllOrderAttributeTypes_shouldGetAllOrderAttributeTypes() {
+		List<OrderAttributeType> orderAttributeTypes = dao.getAllOrderAttributeTypes();
+		assertEquals(orderAttributeTypes.size(), 4);
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderAttributeTypeByUuid(String)}
+	 * @throws Exception
+	 */
+	@Test
+	public void getOrderAttributeTypeByUuid_shouldGetOrderAttributeTypeByUuidGivenTheUuid() {
```

Comment:
Change the name to something like
`getOrderAttributeTypeByUuid_shouldReturnOrderAttributeTypeUsingProvidedUuid()`

The last part of the method should basically tell us what the method should do in detail.